site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -312,6 +312,14 @@ class Project
      */
     private $releases;
 
+    /**
+     * Code quality metrics for the project
+     * @var float
+     *
+     * @ORM\Column(type="float", nullable=true)
+     */
+    private $projectMetrics;
```

Comment:
The name `projectMetrics` indicates multiple items (plural). That's a simple number you want to store here? How about `score`? (the `project` prefix is already in the name of the table)